src/test/regress/expected/failure_mx_metadata_sync.out

```diff
@@ -187,6 +187,12 @@ WARNING:  server closed the connection unexpectedly
 	before or while processing the request.
 CONTEXT:  while executing command on localhost:xxxxx
 WARNING:  connection not open
+CONTEXT:  while executing command on localhost:xxxxx
+WARNING:  connection not open
+CONTEXT:  while executing command on localhost:xxxxx
```

Comment:
Is there some error check missing?

Reply:
I was just looking into these changes. These errors are coming as we try to `ROLLBACK` during the commit hook. So, before this patch, we didn't have these extra commands, hence no such warnings.

As far as I can tell, these are expected. Do you agree with this?

Reply:
I think it's fine.